docs/source/setup/sample.rst

```diff
@@ -227,6 +233,8 @@ from the environments into the respective libraries function argument and return
       ./isaaclab.sh -p source/standalone/workflows/rl_games/train.py --task Isaac-Ant-v0 --headless
       # run script for playing with 32 environments
       ./isaaclab.sh -p source/standalone/workflows/rl_games/play.py --task Isaac-Ant-v0 --num_envs 32 --checkpoint /PATH/TO/model.pth
+      # run script for recording video of a trained agent (requires installing `ffmpeg`)
+      ./isaaclab.sh -p source/standalone/workflows/rl_games/play.py --task Isaac-Ant-v0 --headless --enable_cameras --video --video_length 200
```

Comment:
Since we enable cameras implicitly in the script, we can remove `--enable_cameras` from the instruction?